tests/test_delete_chunk.c

```diff
@@ -0,0 +1,178 @@
+/*
+  Copyright (C) 2020- The Blosc Development Team <blosc@blosc.org>
+  http://blosc.org
+  License: BSD (see LICENSE.txt)
+
+  Creation date: 2020-09-23
```

Comment:
Not the correct creation date.  Hmmm, probably worth getting rid of this line for future files.